src/Lecture1.hs

```diff
@@ -81,7 +88,15 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+
+minmax :: (Num a, Ord a) => a -> a -> a -> a
+minmax x y z = 
+    let large = if x > y then x else y 
+        largest = if large > z then large else z 
+        small = if x < y then x else y 
+        smallest = if small < z then small else z 
```

Comment:
Try using standard functions `max` and `min` to simplify the implementation

Reply:
ok !